docs/data/joy/components/radio-button/ExampleAlignmentButtons.tsx

```diff
@@ -19,6 +19,7 @@ export default function ExampleAlignmentButtons() {
       onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
         setAlignment(event.target.value)
       }
+      sx={{ display: 'inline-flex' }}
```

Comment:
Not ideal to override the `display` property of the `RadioGroup` component (it uses `flex`). In hindsight, `RadioGroup` should've probably used `inline-flex` from the start.

Another option is to wrap the entire demo in an `inline-block/flex` element, but that probably adds more noise to the demo.

Reply:
Perhaps something we can introduce as a breaking change in a future major?